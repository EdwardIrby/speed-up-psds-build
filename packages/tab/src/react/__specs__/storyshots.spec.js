import initStoryshots, {
  snapshotWithOptions
} from '@storybook/addon-storyshots'

jest.mock('@pluralsight/ps-design-system-storybook-addon-theme')

const createNodeMock = el => document.createElement('div')

initStoryshots({
  framework: 'react',
  test: snapshotWithOptions({ createNodeMock })
})
