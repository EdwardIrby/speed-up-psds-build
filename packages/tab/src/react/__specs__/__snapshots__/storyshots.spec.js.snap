// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots default default 1`] = `
<div
  style={
    Object {
      "background": "#181818",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 300ms",
    }
  }
>
  <div>
    <div
      className="css-zhyi0r"
      onKeyDown={[Function]}
      role="tablist"
      tabIndex="0"
    >
      <button
        aria-selected={true}
        className="css-1pp0sr"
        onClick={[Function]}
        role="tab"
        tabIndex="-1"
      >
        <div
          className="css-1vk06pf"
        >
          Menu 1
          <span
            className="css-1s34vsj"
          />
        </div>
      </button>
      <button
        aria-selected={false}
        className="css-1bb8k7d"
        onClick={[Function]}
        role="tab"
        tabIndex="-1"
      >
        <div
          className="css-1vk06pf"
        >
          Menu 2
          <span
            className="css-1s34vsj"
          />
        </div>
      </button>
      <button
        aria-selected={false}
        className="css-1bb8k7d"
        onClick={[Function]}
        role="tab"
        tabIndex="-1"
      >
        <div
          className="css-1vk06pf"
        >
          Menu 3
          <span
            className="css-1s34vsj"
          />
        </div>
      </button>
      <button
        aria-selected={false}
        className="css-1bb8k7d"
        onClick={[Function]}
        role="tab"
        tabIndex="-1"
      >
        <div
          className="css-1vk06pf"
        >
          Menu 4
          <span
            className="css-1s34vsj"
          />
        </div>
      </button>
      <button
        aria-selected={false}
        className="css-1bb8k7d"
        onClick={[Function]}
        role="tab"
        tabIndex="-1"
      >
        <div
          className="css-1vk06pf"
        >
          Menu 5
          <span
            className="css-1s34vsj"
          />
        </div>
      </button>
    </div>
    <div
      aria-labelledby="example3"
      className="css-nil"
      role="tabpanel"
    >
      <div
        className="content"
      >
        Menu stuff 3
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots style overrides list no borderBottom (style) 1`] = `
<div
  style={
    Object {
      "background": "#181818",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 300ms",
    }
  }
>
  <div
    className="css-zhyi0r"
    onKeyDown={[Function]}
    role="tablist"
    style={
      Object {
        "borderBottom": "none",
      }
    }
    tabIndex="0"
  >
    <button
      aria-selected={true}
      className="css-1pp0sr"
      onClick={[Function]}
      role="tab"
      tabIndex="-1"
    >
      <div
        className="css-1vk06pf"
      >
        Wow
        <span
          className="css-1s34vsj"
        />
      </div>
    </button>
  </div>
</div>
`;

exports[`Storyshots style overrides listItem disabled opacity (style) 1`] = `
<div
  style={
    Object {
      "background": "#181818",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 300ms",
    }
  }
>
  <div
    className="css-zhyi0r"
    onKeyDown={[Function]}
    role="tablist"
    tabIndex="0"
  >
    <button
      aria-selected={true}
      className="css-1pp0sr"
      onClick={[Function]}
      role="tab"
      style={
        Object {
          "opacity": 0.5,
        }
      }
      tabIndex="-1"
    >
      <div
        className="css-1vk06pf"
      >
        Disabled look
        <span
          className="css-1s34vsj"
        />
      </div>
    </button>
  </div>
</div>
`;

exports[`Storyshots style overrides panel disabled opacity (style) 1`] = `
<div
  style={
    Object {
      "background": "#181818",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 300ms",
    }
  }
>
  <div>
    <div
      className="css-zhyi0r"
      onKeyDown={[Function]}
      role="tablist"
      tabIndex="0"
    >
      <button
        aria-selected={true}
        className="css-1pp0sr"
        onClick={[Function]}
        role="tab"
        style={
          Object {
            "opacity": 0.5,
          }
        }
        tabIndex="-1"
      >
        <div
          className="css-1vk06pf"
        >
          Disabled look
          <span
            className="css-1s34vsj"
          />
        </div>
      </button>
    </div>
    <div
      aria-labelledby="wow"
      className="css-nil"
      role="tabpanel"
      style={
        Object {
          "border": "1px solid red",
        }
      }
    >
      This is the content
    </div>
  </div>
</div>
`;
